fortifyapi/client.py

```diff
@@ -584,21 +584,34 @@ def get(self, id):
         with self._api as api:
             return Issue(self._api, api.get(f"/api/v1/projectVersions/{self.parent['id']}/issues/{id}")['data'], self.parent)
 
-    def assign(self, user):
+    def assign(self, id, user, revision=0):
         """
-        :param user: user id?
+        :param id: Integer of the issue, typically a 3 to 7 digit int NOT the IID
+        :param user: user login id
+        :param revision: Begins with 0 and incremented with each assignment
```

Comment:
I presume this is something we can actually specify/set, not just an 'internal ssc thing'?

Reply:
yes. however, the user needs to exist in SSC